tslint.json

```diff
@@ -96,8 +96,8 @@
     ],
     "increment-decrement": false,
     "radix": false,
-    "directive-selector": [true, "attribute", ["htc"], "camelCase"],
-    "component-selector": [true, "element", ["htc"], "kebab-case"],
+    "directive-selector": [true, "attribute", ["htc", "test"], "camelCase"],
```

Comment:
rather than changing the lint rules, can we just disable this lint rule as needed? It's rare that we actually define a component in a test.

Reply:
okay